keras_cv/bounding_box/converters.py

```diff
@@ -30,11 +31,16 @@ def _encode_box_to_deltas(
     boxes: tf.Tensor,
     anchor_format: str,
     box_format: str,
-    variance: Optional[List[float]] = None,
+    variance: Optional[Union[List[float], tf.Tensor]] = None,
```

Comment:
is this type annotation useful still or does it become overly burdensome?  No strong feelings, just might be worth dropping it

Reply:
To me, it feels burdensome. But whenever I read someone else's code with type annotation, I still feel it's pretty useful. So it's really a double-edged sword IMHO